src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,28 @@
-import React from 'react';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+interface CommentListProps {
+  comments: Comment[];
+}
+
+export const CommentList: React.FC<CommentListProps> = ({ comments }) => {
+  const isHaveComments = comments.length > 0;
+  const commentsAmmount = comments.length;
```

Comment:
```suggestion
  const commentsAmmount = comments.length;
  const hasComments = commentsAmmount > 0;
```

Reply:
Please fix @Ihor-Karpyn comment and we are done